domain-models-interface-extensions/src/main/java/org/folio/rest/tools/ClientGenerator.java

```diff
@@ -431,7 +431,7 @@ private void encodeParameter(JBlock b, ParameterDetails details) {
   private void formatDateParameter(JBlock b, ParameterDetails details) {
     JExpression expr = jcodeModel.ref(java.time.format.DateTimeFormatter.class)
       .staticInvoke("ofPattern")
-        .arg("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+        .arg("yyyy-MM-dd'T'HH:mm:ss")
```

Comment:
Can you use DateTimeFormatter.ISO_LOCAL_DATE_TIME?
Can you add a unit test to DemoRamlRestTest that has both date and time?

Reply:
Sure and maybe.

Reply:
Done. The additional test will have to suffice. The implementation, BooksDemoAPI,  does not provide any use of the datetime parameters to make any useful assertions.